torch/utils/_cxx_pytree.py

```diff
@@ -182,8 +181,19 @@ def register_pytree_node(
             )
         )
     """
+    from ._pytree import _register_pytree_node
+
+    _register_pytree_node(
+        cls,
+        flatten_func,
+        unflatten_func,
+    )
+
```

Comment:
should _register_pytree_node on python_pytree also register it to cxx_pytree?

Reply:
I think we can do this in the follow-up PR to "make `optree` optional".

Reply:
sgtm